src/ListItem.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { FaAngleUp, FaAngleDown, FaMinus, FaCheck } from 'react-icons/fa'
+
+
+const ListItem = props => {
```

Comment:
one thing you could do is to destructure props. you would have to update the rest of the file.
Just a personal thing that I think looks more descriptive.

```suggestion
const ListItem = ({ handleEdit }) => {
```